src-tauri/src/athan/mod.rs

```diff
@@ -124,9 +124,21 @@ pub fn check_athan_time(store: &mut Store<Wry>, resources_path: PathBuf) {
     let time_remaining = minutes_left.abs_diff(remind_bar_before);
     debug!("Time left before remind: {}", time_remaining);
 
+    // Every minute, check if the current time is the time to remind of prayer
+
+    if hours_left == 0 && time_remaining == 0 {
```

Comment:
Make it `time_remaining <= 1` 7it the function runs every minute, o tkhaf tzgelha.

Reply:
when i made it `time_remaining <= 1` before this change, the function checks every minute, so the entire time between the time it reminds you and the the actual time of prayer, it pushs a notification every single minute